terminal.c

```diff
@@ -6602,7 +6622,26 @@ void term_mouse(Terminal *term, Mouse_Button braw, Mouse_Button bcooked,
               case MA_DRAG:
                 if (term->xterm_mouse == 1)
                     return;
-                encstate += 0x20;
+                encstate += 0x20; // motion indicator
+                break;
+              case MA_MOVE:    //mouse move without buttons
+                assert( braw == MBT_NOTHING && bcooked == MBT_NOTHING  );
+                if (term->xterm_mouse < 3)
+                    return;
+
+                // TODO: 0-based or 1-based ? Store selpoint in Terminal ?
+                if (selpoint.x == term->raw_mouse_reported_x &&
```

Comment:
If you've reached that `return` (when in question add some minor temporarily debugging code) then I think this comment can be dropped as it "works".
As the selpoint is compared I see not where this would be stored in the if - or is it actually what you've thought of doing in the "else" case?